src/contentscript/overlay/root/pages/ConnectedAccount/ConnectedAccount.module.scss

```diff
@@ -179,45 +183,14 @@
-  background-repeat: no-repeat;
-  background-position: center;
-
-  &:hover {
-    background-color: $WebBG;
-  }
-
-  &:active {
-    background-color: $OverlayBG;
-  }
-
-  &:disabled {
-    cursor: default;
-
-    background-color: $PureWhite;
-  }
-}
-
-.accountButtonChoise {
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='7' height='3' viewBox='0 0 7 3' fill='none'%3E%3Cpath d='M5.64307 2.21568C5.24858 2.21568 4.92878 1.89588 4.92878 1.5014C4.92878 1.10691 5.24858 0.787109 5.64307 0.787109C6.03756 0.787109 6.35735 1.10691 6.35735 1.5014C6.35735 1.69084 6.2821 1.87252 6.14814 2.00647C6.01419 2.14043 5.83251 2.21568 5.64307 2.21568ZM3.50021 2.21568C3.10572 2.21568 2.78592 1.89588 2.78592 1.5014C2.78592 1.10691 3.10572 0.787109 3.50021 0.787109C3.8947 0.787109 4.2145 1.10691 4.2145 1.5014C4.2145 1.69084 4.13924 1.87252 4.00529 2.00647C3.87133 2.14043 3.68965 2.21568 3.50021 2.21568ZM1.35735 2.21568C0.962863 2.21568 0.643066 1.89588 0.643066 1.5014C0.643066 1.10691 0.962863 0.787109 1.35735 0.787109C1.75184 0.787109 2.07164 1.10691 2.07164 1.5014C2.07164 1.69084 1.99638 1.87252 1.86243 2.00647C1.72847 2.14043 1.54679 2.21568 1.35735 2.21568Z' fill='%23d9304f'/%3E%3C/svg%3E");
-}
-
 .arrowsAccount {
-  display: inline-block;
+  display: block;
 
   width: 22px;
+  min-width: 22px;
   height: 11px;
-  margin: 0 6px;
 
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='11' viewBox='0 0 16 11' fill='none'%3E%3Cpath d='M0.221076 1.96378L1.63608 0.550781L6.58508 5.50078L1.63008 10.4448L0.222076 9.03678L3.75808 5.50078L0.221076 1.96478L0.221076 1.96378ZM9.41408 5.50078L14.3691 0.555781L15.7781 1.96478L12.2431 5.50078L15.7781 9.03578L14.3641 10.4508L9.41508 5.50178L9.41408 5.50078Z' fill='%23E3E3E3'/%3E%3C/svg%3E");
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='11' viewBox='0 0 16 11' fill='none'%3E%3Cpath d='M0.221076 1.96378L1.63608 0.550781L6.58508 5.50078L1.63008 10.4448L0.222076 9.03678L3.75808 5.50078L0.221076 1.96478L0.221076 1.96378ZM9.41408 5.50078L14.3691 0.555781L15.7781 1.96478L12.2431 5.50078L15.7781 9.03578L14.3641 10.4508L9.41508 5.50178L9.41408 5.50078Z' fill='%23B5B5B5'/%3E%3C/svg%3E");
```

Comment:
Need to think how to separate base64 images to files in the future.

Reply:
https://dapplets-org.atlassian.net/browse/DAP-2192